test/logic/UserAccessTest.scala

```diff
@@ -1,10 +1,9 @@
 package logic
 
-import awscala.DateTime
 import com.gu.googleauth.UserIdentity
 import fixtures.Fixtures._
 import com.gu.janus.model.{ACL, SupportACL}
-import org.joda.time.{DateTimeZone, Period}
+import org.joda.time.{DateTime, DateTimeZone, Period}
```

Comment:
Should we use Java time now instead?

Reply:
Yes we should!  I'm working on that on another branch